addons/website_mass_mailing/controllers/main.py

```diff
@@ -39,12 +40,37 @@ def subscribe(self, list_id, email, **post):
             list_contact_rel.opt_out = False
         # add email to session
         request.session['mass_mailing_email'] = email
+        if post.get('grab_toast_content'):
```

Comment:
Mmh, I don't like this :)

In fact, let's do better:
- Move the `toast_content` field in the mailing list model directly instead of the popup one. This toast content will not be multi-website, that's ok.
- Replace the "alert" of the other newsletter snippet by a toast, with the same content (so this method here always has to return the toast content of the mailing list)

Thank you !